### PR TITLE
Added the functions except log out

### PR DESCRIPTION
**Pull request by PatsiChang**
The log out function need to wait for backend implementation (RECIPE-93)
Link: https://smart-home-cloud.atlassian.net/browse/RECIPE-90

### CODE REVIEW

File: new/app/loginPage/index.tsx [hunk 1]
```diff
@@ -33,8 +32,8 @@ export default function LoginPage() {
     return (
         <>
             <BasicForm onSubmitCallback={submitFunc}>
-                <BasicTextInput name="uid" label="User Id" type="username"/>
-                <BasicTextInput name="password" label="Password" type="password"/>
+                <BasicTextInput name="userId" label="User Id" type="username" />
+                <BasicTextInput name="logInPasswordHashed" label="Password" type="password" />
```
Comment (ChakShingLo) [inline new/app/loginPage/index.tsx]: why it is named logInPasswordHashed? think the value is not hashed yet?

File: new/lib/userSessionApi.tsx [hunk 2]
```diff
@@ -15,23 +18,27 @@ function recoverToken() {
...
 export async function loginWithUidAndPassword(uid: string, password: string): Promise<string> {
-    // TODO mocking login
     console.log("sending to loginWithUidAndPassword: " + uid + " | " + password);
-    await new Promise((r) => setTimeout(r, 1000));
-    if (uid == "test" && password == "test") {
-        sessionToken = "123-123-123";
+    // await new Promise((r) => setTimeout(r, 1000));
+    const person: UserLogin = {
+        userId: uid,
+        logInPasswordHashed: password,
```
Comment (ChakShingLo) [inline new/lib/userSessionApi.tsx:25]: will the password be hashed on client-side or server-side?

Comment (PatsiChang) [reply new/lib/userSessionApi.tsx:25]: you are right, at this moment its not hashed, but i think we will hash it before we sent it to the server by encryption?

File: new/app/loginPage/index.tsx (hunk 1 above)
Comment (PatsiChang) [reply new/app/loginPage/index.tsx]: This is fixed

**Review by ChakShingLo — APPROVED** [review]

**Pull request merged**